docs/getting-started/quickstart.md

```diff
@@ -355,6 +365,41 @@ pprint(feature_vector)
 {% endtab %}
 {% endtabs %}
 
+## Step 7: Using a featureservice to fetch online features instead.
+
+You can also use the feature service to manage multiple features and be a central source of truth for a series of feature views. The feature store can also be used to fetch either online or historical features using the same api below. More information can be found [here](https://docs.feast.dev/getting-started/concepts/feature-service).
+
+{% tabs %}
+{% tab title="Python" %}
+```python
+from feast import FeatureStore
+feature_store = FeatureStore('.')  # Initialize the feature store
+
+feature_service = feature_store.get_feature_service("driver_activity")
+features = feature_store.get_online_features(
+    features=feature_service,
+    entity_rows=[
+        # {join_key: entity_value}
+        {"driver_id": 1004},
+        {"driver_id": 1005},
+    ],
+).to_dict()
+```
+
+{% tabs %}
+{% tab title="Output" %}
+```bash
+{
+ 'acc_rate': [0.5732735991477966, 0.7828438878059387],
+ 'avg_daily_trips': [33, 984],
+ 'conv_rate': [0.15498852729797363, 0.6263588070869446],
+ 'driver_id': [1004, 1005]
```

Comment:
Do we expect these exact values? If not, we should caveat that the numbers here in the example may be different from what users see.

Reply:
Done.